op-challenger/game/fault/types/position.go

```diff
@@ -81,11 +122,23 @@ func (p Position) Defend() Position {
 }
 
 func (p Position) Print(maxDepth int) {
-	fmt.Printf("GIN: %4b\tTrace Position is %4b\tTrace Depth is: %d\tTrace Index is: %d\n", p.ToGIndex(), p.indexAtDepth, p.depth, p.TraceIndex(maxDepth))
+	fmt.Printf("GIN: %4b\tTrace Position is %4b\tTrace Depth is: %d\tTrace Index is: %d\n", p.ToGIndex(), p.indexAtDepth.Uint64(), p.depth, p.TraceIndex(maxDepth))
+}
+
+func (p Position) ToGIndex() *big.Int {
+	return big.NewInt(0).Or(big.NewInt(0).Lsh(big.NewInt(1), uint(p.depth)), p.IndexAtDepth())
```

Comment:
```suggestion
	return new(big.Int).Or(new(big.Int).Lsh(big.NewInt(1), uint(p.depth)), p.IndexAtDepth())
```

The `new(big.Int)` syntax is a little more typically but also avoids having `0` in the equation which looks like it's important when its actually completely ignored and just being used as the final target for the result.